Urlopik/.idea/dictionaries/Marcin.xml

```diff
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
```

Comment:
zignoruj i wyrzuc z commita cały folder .idea - to sa pliki konfiguracyjne narzędzia - nie trzyma się tego w repozytorium